react/src/js/components/Consonant/Cards/Card.jsx

```diff
@@ -399,7 +402,7 @@ const Card = (props) => {
     const altCtaLink = getAltCtaLink(footer);
     const ctaText = (altCtaUsed && isUpcoming && altCtaLink !== '') ? getCtaText(footer, 'alt') : getCtaText(footer, 'right');
     const overlay = (altCtaUsed && isLive && altCtaLink !== '') ? altCtaLink : overlayParams;
-    const getsFocus = (isHalfHeight && !videoURLToUse)
+    const getsFocus = isHalfHeight

```

Comment:
Since we removed `&& !videoURLToUse` just double checking that cards with videos shouldn’t be focusable?